front/src/modules/settings/workspace/components/NewObjectType.tsx

```diff
@@ -0,0 +1,156 @@
+import React, { useState } from 'react';
+import { useTheme } from '@emotion/react';
+import styled from '@emotion/styled';
+
+import { IconBox, IconCheck, IconDatabase, IconFileCheck } from '@/ui/icon';
+import { SoonPill } from '@/ui/pill/components/SoonPill';
+import { Tag } from '@/ui/tag/components/Tag';
+import { ThemeColor } from '@/ui/theme/constants/colors';
+
+const StyledContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  gap: ${({ theme }) => theme.spacing(5)};
+  padding-bottom: ${({ theme }) => theme.spacing(10)};
+  padding-top: ${({ theme }) => theme.spacing(4)};
+  width: 100%;
+  & > div {
+    flex: 1;
+  }
```

Comment:
Let's remove this and set the `StyledObjectTypeCard` width to `139.33px` instead.

Reply:
Let's use 140px, we don't need it to be 139.33px